components/function-controller/pkg/apis/serverless/v1alpha2/function_defaults_test.go

```diff
@@ -110,6 +112,7 @@ func TestSetDefaults(t *testing.T) {
 						MinReplicas: &two,
 						MaxReplicas: &two,
 					},
+					Replicas: &two,
```

Comment:
Generally we can remove scaleConfig with two equal values in cases where we don't check it.

Reply:
I this is one also is related to the [defaulting and validation](https://github.com/kyma-project/kyma/issues/14900) issue.